### DISCUSSION TITLE
[Request] User-friendly printing of information in manifest (vcpkg.json)

### DISCUSSION THREAD

**Discussion by VelocityRa** [microsoft/vcpkg#17938] [Ideas]
There's currently no way for the user to see optional features, their descriptions, etc without reading the `vcpkg.json`.

**Comment by NancyLi1013** [comment]
Hi @VelocityRa 

Currently, you can get these information from [vcpkg website](https://vcpkg.io/en/index.html). Could you please help confirm if this can meet your requirements?

You can get access to vcpkg website, select **packages** item to see the information for the ports. If you want to see the features, just click `View Details` -> `Feature List`.

Please see the result:

![image](https://user-images.githubusercontent.com/46708020/118241017-3cb9bc00-b4ce-11eb-917f-261e5a5aa5d7.png)

**Comment by VelocityRa** [comment]
No this issue is about `vcpkg.json`. Project-specific manifests.
